docs/integrations/saas-cloud/armis.md

```diff
@@ -0,0 +1,233 @@
+| count_distinct(id)
+```
+
+## Collecting Logs for Armis Api
+
+This section explains how to collect logs from Armis Api and ingest them into Sumo Logic. Refer to the [Armis Api Cloud-to-Cloud Integration](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/armis-api-source/) to create the source and use the same source category while installing the app.
+
+## Installing the Armis App
+
+Locate and install the app from the **App Catalog**. If you want to see a preview of the dashboards included with the app before installing, click **Preview Dashboards**.
+
+To install the app, follow the steps below:
+1. From the **App Catalog**, search for the app and select it.
+2. Select the version of the service you're using and click **Add to Library**.
+
+:::note
+Version selection is applicable only to a few apps currently. For details, see the [Install the Apps from the Library](/docs/get-started/apps-integrations#install-apps-from-the-library).
+:::
+3. To install the app, enter the following fields.
+   1. **App Name**. You can retain the existing name, or enter a name of your choice for the app.
+   2. **Data Source**. Select either of these options for the data source:
+      * Choose **Source Category** and then choose a source category from the list.
+      * Select **Enter a Custom Data Filter** and type in a custom source category that starts with an underscore. For Example, `_sourceCategory=MyCategory`.
+   3. **Advanced**. Select the **Location in Library** (the default is the **Personal** folder in the library), or click **New Folder** to add a new folder.
+4. Click **Add to Library**.
+
+Once an app is installed, it will appear in your **Personal** folder, or other folder that you specified. You can share it with your organization.
+
+The panels will begin to fill automatically. It's worth noting that each panel gradually fills with data that matches the time range query and has been received since the panel was created. The results will not be available right away, but with some patience, you will be able to view full graphs and maps.
+
+## Viewing the Armis Dashboards​
+
+* All dashboard have a set of filters that you can apply to the entire dashboard, as shown in the following example. Click the funnel icon in the top dashboard menu bar to display a scrollable list of filters that are applied across the entire dashboard.
+
+ You can use filters to drill down and examine the data on a granular level. Filters include client country, client device type, client IP, client request host, client request URI, client request user agent, edge response status, origin IP, and origin response status.
+
+* Each panel has a set of filters that are applied to the results for that panel only, as shown in the following example. Click the funnel icon in the top panel menu bar to display a list of panel-specific filters.
+
+
+## Alerts Overview Dashboard
```

Comment:
```suggestion
Subsection of "Viewing Dashboards"
### Alerts Overview Dashboard
```

Reply:
> 
Thanks, kim for the suggestion. I belive we don't need to mention that this is a subsection as i have used the third heading.

Reply:
The heading you've used is the second (H2) heading (`## Alerts Overview Dashboard`). Should be third (H3).

Reply:
Yeas that's correct. Thank you for pointing out this one.

Reply:
I see the change now - just approved!